Firestore/core/test/firebase/firestore/testutil/testutil.h

```diff
@@ -131,6 +138,39 @@ inline std::unique_ptr<model::SetMutation> SetMutation(
       model::Precondition::None());
 }
 
+inline std::unique_ptr<model::PatchMutation> PatchMutation(
```

Comment:
The logic is not that simple and I assume it is used in a few places. How about remove inline and save compilation time?

Reply:
Done.